Modules/TOF/src/TaskRaw.cxx

```diff
@@ -47,27 +47,16 @@ void RawDataDecoder::decode()
 
 void RawDataDecoder::headerHandler(const CrateHeader_t* crateHeader, const CrateOrbit_t* crateOrbit)
 {
-  // DRM Counter
-  if (crateHeader->slotPartMask & 0 << 0) {
-    mDRMCounter[crateHeader->drmID].Count(0);
-  }
-
-  // LTM Counter
-  if (crateHeader->slotPartMask & 1 << 0) {
-    mLTMCounter[crateHeader->drmID].Count(0);
-  }
-
-  // TRM Counter
-  for (int i = 1; i < 11; i++) {
-    if (crateHeader->slotPartMask & 1 << i) {
-      mTRMCounter[crateHeader->drmID][i - 1].Count(0);
-    }
-  }
-
-  // Paricipating slot
-  for (int ibit = 0; ibit < 11; ++ibit) {
+  //Participating slot
+  for (int ibit = 0; ibit < 11; ++ibit) {    
     if (crateHeader->slotPartMask & (1 << ibit)) {
-      mSlotPartMask->Fill(crateHeader->drmID, ibit + 2);
+      // DRM Counter
+      mDRMCounter[crateHeader->drmID].Count(0);
+      mSlotPartMask->Fill(crateHeader->drmID, ibit + 1);
```

Comment:
Why using `ibit + 1` instead of `ibit + 2` ?

Reply:
Since in the SlotPartMask there is no bit for DRM I thought to move the indices, but you probably wanted to be consisted with the SlotID variable right? In this case I can change it back makes sense